### DISCUSSION TITLE
Ternary inside sx prop

### DISCUSSION THREAD

**Discussion by Cody-Cooper** [system-ui/theme-ui#2202] [Q&A]
I'm having a problem on my gatsby app where using a ternary operation inside the sx prop will work just fine in my dev server but doesn't work at all in production/after building. Has anyone had to deal with this before?

example: 
` <section
          sx={{
            top: rootPage
              ? "50px"
              : "25px",
          }}
        >`

**Comment by hasparus** [comment]
Could you provide a small reproduction?
